Makefile

```diff
@@ -180,7 +183,7 @@ gotestdashi: $(C_LIBS) $(CGO_FLAGS_FILES) $(BOOTSTRAP_TARGET)
 
 testshort: override TESTFLAGS += -short
 
-testrace: override GOFLAGS += -race
+testrace: override GOFLAGS += -race ## Run tests with the Go race detector enabled.
```

Comment:
I think I'd prefer if this were on its own line, like

```bash
testrace: ## Run tests with the Go race detector enabled
```

to minimize diff noise.